src/java/org/apache/cassandra/io/sstable/format/SSTableWriter.java

```diff
@@ -439,7 +441,8 @@ public static void hardlink(Descriptor tmpdesc, Descriptor newdesc, Set<Componen
 
     public static void guardCollectionSize(TableMetadata metadata, DecoratedKey partitionKey, Unfiltered unfiltered)
     {
-        if (!unfiltered.isRow() || SchemaConstants.isInternalKeyspace(metadata.keyspace))
+//        if (!unfiltered.isRow() || SchemaConstants.isInternalKeyspace(metadata.keyspace))
```

Comment:
Don't leave commented out code. If it is not needed, remove it. The history of changes is stored in git anyway, so you can always go back and nothing is lost.